src/parsers/pulldown.rs

```diff
@@ -325,25 +341,26 @@ impl CommonMarkViewerInternal {
             scroll_cache.split_points.clear();
         }
     }
-
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Clippy complained about too many arguments.
Unsure if I could somehow reduce the number of arguments, so I just added an allow